test/kokoro/xds.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -exu -o pipefail
+if [[ -f /VERSION ]]; then
+  cat /VERSION
+fi
+
+cd github
+
+export GOPATH=${HOME}/gopath
+pushd grpc-go/interop/xds/client
+go build
+popd
```

Comment:
Can you not do `go build ./grpc-go/interop/xds/client` (because of modules checking the current directory?)?

Reply:
Build probably doesn't work outside of the grpc-go directory, because it cannot find `go.mod`

Reply:
This doesn't work from the parent directory of `grpc-go` locally. I think it would have to be:

```
pushd grpc-go
go build ./interop/xds/client
popd
```

Which I've updated it to. As the PR history here shows, I'm not at all familiar with building go so if this can be simplified further let me know.